Parsers/ASimAuditEvent/Parsers/vimAuditEventCiscoMeraki.yaml

```diff
@@ -0,0 +1,260 @@
+Parser:
+  Title: Audit Event ASIM parser for Cisco Meraki
+  Version: '0.1'
+  LastUpdated: Jun 20 2023
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: Cisco Meraki Documentation
+  Link: https://documentation.meraki.com/
```

Comment:
can we add direct link of the page which contain Audit logs related info?

Reply:
Done